homeassistant/components/device_automation/__init__.py

```diff
@@ -180,6 +185,23 @@ async def _async_get_device_automation_capabilities(hass, automation_type, autom
     return capabilities
 
 
+def get_supported_features(hass: HomeAssistant, entity_id: str) -> int:
```

Comment:
Not sure where this helper should be placed? Maybe in entity_registry or helpers/entity.py instead?

Reply:
`helpers/entity.py` seems like the right place.